__tests__/qraphql_queries/user.test.js

```diff
@@ -25,8 +25,8 @@ describe("test validity of GraphQL queries and mutations", () => {
 
     describe("register mutation", () => {
         const register_mutation = `
-            mutation Register ($user: String!, $pass: String!){
-                register(username: $user, password: $pass){
+            mutation Register ($user: String!, $email: String!, $pass: String!){
+                register(username: $user, email: $email, password: $pass){
```

Comment:
fixed test to account for changes in register mutation

Reply:
Very good, I wanted to mention it during our next meeting 👍